sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -164,11 +164,15 @@ private void apiConnectionLogic(HttpServletRequest request, HttpServletResponse
 			clientRequest.addHeader(h.getName(), h.getValue());
 		}
 		BufferedReader rd = null;
+		long startTime = System.currentTimeMillis();
 		try {
 			rd = new BufferedReader(new InputStreamReader(client.execute(clientRequest).getEntity().getContent()));
 		} catch (IOException e) {
 			M_log.error("Canvas API call did not happen",e);
```

Comment:
A more accurate comment would be something like: "Canvas API call did not complete successfully".